starter-code/models/place.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const placeSchema = new Schema({
+  name: String,
+  longitude: Number,
+  latitude: Number
```

Comment:
When you want to store coordinates with mongoose as GeoJSON (recommended) you do:
```
location: { 
  type: { type: String, default:'Point' }, 
  coordinates: [Number]
}
```